### PR TITLE
[new release] xcursor (0.1.0)

### PR DESCRIPTION
**Pull request by yilinwei**
Xcursor

- Project page: <a href="https://github.com/yilinwei/ocaml-xcursor">https://github.com/yilinwei/ocaml-xcursor</a>
- Documentation: <a href="https://yilinwei.github.io/ocaml-xcursor">https://yilinwei.github.io/ocaml-xcursor</a>

##### CHANGES:

Initial release.

### CODE REVIEW

**Comment by yilinwei** [comment]
Would it be possible to wait until I've regenerated the `.opam` file in the main repository and artifacts to be the same as this one please? Otherwise it will be confusing.

Should I open it as a separate PR?

**Comment by mseri** [comment]
No need to create a new PR, you can even force push here if you want

**Comment by mseri** [comment]
Let me know when I can merge, this looks good to me.

By the way, opam will use the opam file from this repository as source of truth, in general with time upper bounds are added and they go out of sync. As long as the dev branch is kept updated, you don't need to regenerate the release unless you need to modify the code

**Comment by yilinwei** [comment]
@mseri 

Should be good to go now, thanks!

File: packages/xcursor/xcursor.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Xcursor"
+description: "A pure implementation of Xcursor in Ocaml"
```
Comment (kit-ty-kate) [inline packages/xcursor/xcursor.0.1.0/opam:3]: a synopsis that doesn’t tell anything isn’t really useful. The description you had looks more like a synopsis than a description so you can just make this one the synopsis:
```suggestion
synopsis: "A pure implementation of Xcursor in Ocaml"
```

Comment (yilinwei) [reply packages/xcursor/xcursor.0.1.0/opam]: Thanks! I will take a look at some better descriptions and update accordingly.

Comment (yilinwei) [reply packages/xcursor/xcursor.0.1.0/opam]: Does the new description fit the criteria?

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**